### DISCUSSION TITLE
Global Configuration for Component Settings

### DISCUSSION THREAD

**Discussion by DinoCloudlet** [primefaces/community#1823] [PrimeNG]
It would be nice to be able to set certain settings globally so that there is no need to define the inputs every time a component is used. This leads to a consistent user experience. The setting i am mostly interested is the tooltip Delay.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for the valuable feedback. We'll wait for the upvotes to implement this as a feature.
